lib/hawkular/websocket_client.rb

```diff
@@ -0,0 +1,82 @@
+require 'event_emitter'
+require 'eventmachine'
+require 'faye/websocket'
+
+module Hawkular
+  class WebsocketClient
+    include EventEmitter
+    @channel = EventMachine::Channel.new
+    attr_accessor :url, :ws, :open, :options, :thread
+    attr_reader :open
+    alias_method :open?, :open
+    class << self
+      attr_accessor :channel
+    end
+
+    def self.running_evm
```

Comment:
I don't like this method name, if it starts EventMachine, we should call it `#run` or something like that. We don't need to make any references to EventMachine, since it is also an implementation detail.